nifi-extension-bundles/nifi-poi-bundle/nifi-poi-services/src/main/java/org/apache/nifi/processors/excel/SplitExcel.java

```diff
@@ -229,7 +217,56 @@ public void onTrigger(ProcessContext context, ProcessSession session) throws Pro
         session.transfer(flowFileSplits, REL_SPLIT);
     }
 
-    private int copyRows(final Sheet originalSheet, final SXSSFSheet destinationSheet) {
+    private CellCopyPolicy createCellCopyPolicy(InputFileType inputFileType) {
```

Comment:
Instead of creating a new `CellCopyPolicy` for every FlowFile processed, can two different implementation instances be created statically and shared?